src/templates/polykart.ts

```diff
@@ -101,7 +54,7 @@ export function getGenerator(options?) {
         return githubToken;
```

Comment:
I'd like you to move this function to github and use it as a default value for Github()